### ISSUE TITLE
Crash v1.7.5 (Fedora miscompilation)

### ISSUE THREAD

**Issue by mariobl** [vslavik/poedit#153]
There's a crash report in Redhat Bugzilla, regarding the Fedora 22 package.
See https://bugzilla.redhat.com/show_bug.cgi?id=1202572
The report is automatically generated. As far as I can see, it obviously refers to wxGTK, not to Poedit itself. On Fedora 22, Poedit has been build with wxgtk-3.0.2, lucene++-3.0.6 and boost-1.57.0. Please have a look at the backtrace attached to the Bugzilla ticket.

**Comment by vslavik** [comment]
> As far as I can see, it obviously refers to wxGTK, not to Poedit 

Then why the heck do you feel the need to post it here, when you *know* it’s not a Poedit issue?! This makes no sense.

> Please have a look at the backtrace attached to the Bugzilla ticket.

Please kindly filter out Fedora's own crap before wasting upstream’s time. The least you can do is to have a look at your automated stuff *yourself* first to determine it’s validity.

If you as much as *looked* at the backtrace — which you ask upstream to do — you would immediately see that you guys managed to screw up compilation of the packages. How, do you ask, could you possibly tell, perhaps not being sufficiently familiar with Poedit and/or wxWidgets? Why, *because it tells  you so right there* and in plain English no less! See:

```
Mismatch between the program and library build versions detected.
```

(followed by detailed description of the cause, shown at runtime by the program, but truncated in the log).

**Comment by andreasstieger** [comment]
Ugh, Fedora's (re)build system needs fixing.

**Comment by mariobl** [comment]
»Please kindly filter out Fedora's own crap before wasting upstream’s time. The least you can do is to have a look at your automated stuff yourself first to determine it’s validity.«

Sorry for the blurb, but I'm just a packager, I don't have any coding skills. It was only a presumption that it could be caused by wxgtk.
